module4/projects/mod4Project/ideation.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Ideation
+---
+
+<section class='instructor-notes'>
+
+In order to cut down on the time groups could spend deciding on a project, we will use a most-popular vote system to determine the projects, then students will 'opt-in' to projects based on their preference.
+
+It may not be necessary for _every_ student to bring an idea, but we should have at least 1 project pitch for each 3/4 students.
```

Comment:
I think it might be a good idea to have each student pitch something. Especially because we have an hour for pitch prep, I want everyone to do something. And then they could have 3 minutes to pitch to the class and we have a form for voting on your top 4 project picks or something like that.